praetorian_cli/sdk/entities/risks.py

```
@@ -85,20 +85,25 @@ def delete(self, key, status, comment=None):
 
         return self.api.delete_by_key('risk', key, body)
 
-    def list(self, prefix_filter='', offset=None, pages=100000) -> tuple:
+    def list(self, contains_filter='', offset=None, pages=100000) -> tuple:
```

Ruff (0.14.0)

Unused method argument: `offset`

(ARG002)